variable/energy/energy-final.yaml

```diff
@@ -273,3 +273,38 @@ Final Energy|Transportation|Liquids|Oil:
    description: Final energy consumption by the transportation sector of liquid
                 oil products (from conventional & unconventional oil)
    unit: EJ/yr
+
+Final Energy|Electricity|Profile:
```

Comment:
I'm wondering why this needs an own variable? If the variable `Final Energy` is given for a full year, it is the total consumption over the year - if it is given with an hourly disaggregation, it already fulfils the role of this "profile". So I see this as a duplication.